### PR TITLE
add gaussian-view

### PR DESCRIPTION
**Pull request by asmaahassan90**
Add a new package (gaussian-view)

### CODE REVIEW

File: var/spack/repos/builtin/packages/gaussian-view/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        install_tree(os.getcwd(), self.prefix)
+
+    def setup_run_environment(self, env):
+        env.set('GV_DIR', self.prefix)
+        env.prepend_path('PATH', self.prefix)
+        env.set('GV_LIB_PATH', self.prefix.lib)
+        env.prepend_path('GV_LIB_PATH', self.prefix.lib.MesaGL)
+        env.prepend_path('LD_LIBRARY_PATH', self.prefix.lib.MesaGL)
+        env.set('ALLOWINDIRECT', '1')
+        env.set('QT_PLUGIN_PATH', self.prefix.plugins)
```
Comment (becker33) [inline var/spack/repos/builtin/packages/gaussian-view/package.py]: @asmaahassan90 Should this be `env.set` or `env.prepend_path`?

Comment (asmaahassan90) [reply var/spack/repos/builtin/packages/gaussian-view/package.py]: Both. I'm add two paths to `GV_LIB_PATH`. One is `self.prefix.lib` and the other is `self.prefix.lib.MesaGL`. 
Is there a better way to do it?

Comment (asmaahassan90) [reply var/spack/repos/builtin/packages/gaussian-view/package.py]: @becker33 Do you need me to change something here?

Comment (becker33) [reply var/spack/repos/builtin/packages/gaussian-view/package.py]: @asmaahassan90 yes, please change the following:

```suggestion
        env.prepend_path('QT_PLUGIN_PATH', self.prefix.plugins)
```

Comment (becker33) [reply var/spack/repos/builtin/packages/gaussian-view/package.py]: Sorry for the confusion, I wasn't confident in how QT_PLUGIN_PATH worked when I was first reviewing it.

Comment (asmaahassan90) [reply var/spack/repos/builtin/packages/gaussian-view/package.py]: done

**Review by becker33 — APPROVED** [review]

**Pull request merged**